.github/workflows/.jira_sync_config.yaml

```diff
@@ -0,0 +1,22 @@
+# Sync GitHub issues to Jira issues
+
+# Configuration syntax:
+# https://github.com/canonical/gh-jira-sync-bot/blob/main/README.md#client-side-configuration
+settings:
+  # Repository specific settings
+  components: # Jira components that will be added to Jira issue
+    - etcd-vm
+    - snap
```

Comment:
The `snap` component doesn't seem to be used a lot in Jira currently, let's only use the `etcd-vm` component instead.